040-SDK/130-transaction.mdx

```diff
@@ -266,50 +266,58 @@ const result = await xata.transactions.run([
       fields: { email: 'trinity@matrix.com', name: 'Trinity' }
     }
   }
+  // Additional transactions can be added here
 ]);
 ```
 
 ```python
-result = xata.transactions.run([
-  {
-    "update": {
-      "table": "Users",
-      "id": "rec_cmf56ugvjdkilkpcdeu0",
-      "fields": {"email": "neo@matrix.com", "name": "Neo"}
-    }
-  },
-  {
-    "update": {
-      "table": "Users",
-      "id": "rec_cmf572veoo9ahn3r96p0",
-      # Include the fields to update for this record
-    }
-  },
-  # Additional transactions can be added here
-])
+result = xata.records().transactions({
```

Comment:
@philkra should be `transaction` not transactions right?

`xata.records().transaction` from https://github.com/xataio/xata-py/blob/main/xata/api/records.py#L34

Reply:
Oh yikes! yes it should, let me open a PR

Reply:
if this is not already open, I'll just commit it in the main PR. I have to push other changes as well

Reply:
https://github.com/xataio/mdx-docs/pull/226